src/Parallel/GlobalCache.hpp

```diff
@@ -37,8 +42,9 @@ struct type_for_get_helper<std::unique_ptr<T, D>> {
 // an unknown tag is requested from the GlobalCache.
 template <typename GlobalCacheTag, typename ListOfPossibleTags>
 struct list_of_matching_tags_helper {
-  using type = tmpl::filter<ListOfPossibleTags,
-               std::is_base_of<tmpl::pin<GlobalCacheTag>, tmpl::_1>>;
+  using type =
+      tmpl::filter<ListOfPossibleTags,
```

Comment:
I'm not sure if you run clang format on the entire file or why there are so many white space changes. `git clang-format -f` works nicely and only formats lines you actually touch. This makes the review a lot easier so I don't spend time trying to figure out what changed when the answer is "nothing"